src/components/Nav/Nav.scss

```
@@ -0,0 +1,114 @@
+@import '/src/styles/mixin.scss';
+@import '/src/styles/variables.scss';
+
+.nav {
+  position: fixed;
+  width: 100%;
+  min-width: 1050px;
+
+  .header {
+    min-width: 1050px;
+    padding: 0px 15%;
+
+    .signUpLoginCs {
+      @include flexSort(flex-end, null);
+
+      ul {
+        li {
+          float: left;
+          padding: 15px;
+          font-size: $small;
+          font-weight: 400;
+
+          &:nth-child(2) {
+            color: #999999;
+          }
+
+          &:nth-child(3) {
+            color: #999999;
+          }
+
+          .link {
+            text-decoration: none;
+            &:focus,
+            &:hover,
+            &:visited,
+            &:link,
+            &:active {
+              color: black;
+              text-decoration: none;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .main {
+    margin: 20px 0;
+    @include flexSort(space-around, center);
```

```suggestion
    @include flexSort(space-around, center);
    margin: 20px 0;
```

include는 컨벤션상 가장 위에 위치하는 것이 좋습니다.